addons/test_mass_mailing/tests/test_blacklist_behavior.py

```diff
@@ -51,8 +51,8 @@ def test_mailing_bounce_w_auto_bl_partner_duplicates(self):
 
     @mute_logger('odoo.addons.mail.models.mail_thread')
     def _test_mailing_bounce_w_auto_bl(self, bounce_base_values):
-        mailing = self.env['mailing.mailing'].browse(self.mailing_bl.ids)
-        target = self.env['mailing.test.blacklist'].browse(self.target_rec.ids)
+        mailing = self.mailing_bl.with_env(self.env)
```

Comment:
Does that change anything ?

Reply:
Not really, it was a bit of cleaning as I was trying to understand what was occurring, and then I forgot to roll it back.